src/Documentation/Installation/NuGets.md

```diff
@@ -48,94 +48,130 @@ Includes everything you need for an Orleans client (frontend).
 
 The below packages provide additional functionality.
 
-### [Microsoft.Orleans.OrleansServiceBus](http://www.nuget.org/packages/Microsoft.Orleans.OrleansServiceBus/)
+## Providers and extensions
+
+### [Microsoft Orleans Azure Utilities](http://www.nuget.org/packages/Microsoft.Orleans.OrleansAzureUtils/)
+
+```
+PM> Install-Package Microsoft.Orleans.OrleansAzureUtils
+```
+Contains Azure Table based cluster membership provider, wrapper classes that simplifies instantiation of silos and clients in Azure Worker/Web roles, persistence providers for Azure Tables and Azure Blobs, and a stream providers for Azure Queues.
```

Comment:
"Blobs and a stream provider for Azure Queues", extra comma and providerS has an extra s.